apsuite/optimization/base.py

```diff
@@ -211,8 +211,11 @@ def __init__(self, params, use_thread=True, isonline=True):
         self.use_thread = use_thread
 
         self._num_objective_evals = 0
-        self.best_positions = _np.array([], ndmin=2)
-        self.best_objfuncs = _np.array([], ndmin=2)
+        self.positions_evaluated = []
+        self.positions_best = []
+        self.objfuncs_evaluated = []
+        self.objfuncs_best = []
+        self.idcs_cumulated_optimum = None
```

Comment:
Maybe, to keep the standardization of attributes names, this attribute could be called: `objfuncs_cumul_optima_idcs`. What do you think?

Reply:
Done! Also renamed other methods related to "cumulative optima"